### PR TITLE
Update header for georchestra and update login page

### PR DESCRIPTION
**Pull request by f-necas**
This PR changes the login page to better integrate with geOrchetsra new stack. 

It provides several configuration options :

# headerEnabled

If header is enabled with `georchestra.gateway.headerEnabled` in `gateway.yaml`, provides header and hide logo in right column:

image

If disabled :

image

# Icons 

Extenal-auth icons tries to fetch images in `static/login/img/<provider>.png>`. If not available, render an external icon :
image

# Additional configuration 

Additional configuration is possible through the georchestraStylesheet in `default.properties`
`georchestraStylesheet=https://mydomain.fr/public/georchestra.css`

If a css is provided, you can edit style and colors...

### CODE REVIEW

**Comment by fvanderbiest** [comment]
Awesome !

File: gateway/src/main/resources/messages/login_en.properties [hunk 1]
```diff
@@ -10,3 +10,7 @@ expired_password = Your password has been expired
 expired_password_link =  and should be changed
 invalid_credentials = Invalid username or password
 duplicate_account = An account already exists using this email address
+separator_login=Or, Login with
+register_no_account=Don't have an account?
+login_message_subtitle=Discover all the data of your platform
```
Comment (fvanderbiest) [inline gateway/src/main/resources/messages/login_en.properties:15]: "Discover all the datasets on your platform"

File: gateway/src/main/resources/messages/login.properties [hunk 2]
```diff
@@ -10,3 +10,7 @@ expired_password = Your password has been expired
 expired_password_link = and should be changed
 invalid_credentials = Invalid username or password
 duplicate_account = An account already exists using this email address
+separator_login=Or, Login with
+register_no_account=Don't have an account?
+login_message_subtitle=Discover all the data of your platform
```
Comment (fvanderbiest) [inline gateway/src/main/resources/messages/login.properties:15]: "Discover all the datasets on your platform"

File: gateway/src/main/resources/messages/login_de.properties [hunk 3]
```diff
@@ -10,3 +10,7 @@ expired_password = Ihr Passwort ist abgelaufen
 expired_password_link = und sollte geändert werden
 invalid_credentials = Ungültiger Benutzername oder Passwort
 duplicate_account = Es existiert bereits ein Konto mit dieser E-Mail-Adresse
+separator_login=Oder, Anmelden mit
+register_no_account=Sie haben noch kein Konto?
+login_message_subtitle=Entdecken Sie alle Daten Ihrer Plattform
```
Comment (fvanderbiest) [inline gateway/src/main/resources/messages/login_de.properties:15]: Entdecken Sie alle Datensätze auf Ihrer Plattform

**Review by fvanderbiest — APPROVED** [review]

File: gateway/src/main/resources/messages/login_fr.properties [hunk 4]
```diff
@@ -10,3 +10,7 @@ expired_password = Votre mot de passe a expiré
 expired_password_link = et doit être changé
 invalid_credentials = Nom d'utilisateur ou mot de passe non valide
 duplicate_account = Il existe déjà un compte utilisant cette adresse e-mail
+separator_login=Ou, Connectez-vous avec
+register_no_account=Vous n'avez pas de compte?
+login_message_subtitle=Découvrez toutes les données de votre plateforme
```
Comment (fvanderbiest) [inline gateway/src/main/resources/messages/login_fr.properties:15]: Découvrez tous les jeux de données

**Review by fvanderbiest — APPROVED** [review]

File: pom.xml [hunk 5]
```diff
@@ -18,7 +18,7 @@
   </modules>
   <properties>
     <revision>1.1.1-SNAPSHOT</revision>
-    <georchestra.version>24.0.1-SNAPSHOT</georchestra.version>
+    <georchestra.version>24.0.4-SNAPSHOT</georchestra.version>
```
Comment (pmauduit) [inline pom.xml:21]: 24.1-SNAPSHOT ? https://github.com/georchestra/georchestra/blob/master/pom.xml#L18

**Review by pmauduit — COMMENTED** [review]
just surprised by the georchestra.version, if we consider that `main` here reflects `master` on georchestra/georchestra, otherwise, looks good

**Pull request merged**